src/main/java/tds/content/controllers/ContentController.java

```diff
@@ -111,6 +111,13 @@ public ContentController(final ContentService contentService) {
             return ResponseEntity.ok(contentService.loadData(new URI(itemPath)));
         } catch (URISyntaxException e) {
             throw new IllegalArgumentException(String.format("The provided item path '%s' was malformed", itemPath));
+        } catch (final AmazonS3Exception ex) {
```

Comment:
I don't think that our controller's should be handling data access Exceptions.  This allows implementation details leak across layers.  I think we should use the NotFoundException and possible add a new Exception to handle when access cannot be granted to an item.  That would be consistent regardless if the item is on S3 or the file system.